src/coreclr/build-test.cmd

```diff
@@ -616,13 +616,17 @@ set __TotalPrecompiled=0
 set __FailedToPrecompile=0
 set __FailedAssemblies=
 for %%F in ("%CORE_ROOT%\System.*.dll";"%CORE_ROOT%\Microsoft.*.dll") do (
-    if not "%%~nxF"=="Microsoft.CodeAnalysis.VisualBasic.dll" (
-    if not "%%~nxF"=="Microsoft.CodeAnalysis.CSharp.dll" (
-    if not "%%~nxF"=="Microsoft.CodeAnalysis.dll" (
-    if not "%%~nxF"=="System.Runtime.WindowsRuntime.dll" (
-        call :PrecompileAssembly "%%F" %%~nxF __TotalPrecompiled __FailedToPrecompile __FailedAssemblies
-        echo Processed: !__TotalPrecompiled!, failed !__FailedToPrecompile!
-    )))))
+    if "%%~nxF"=="Microsoft.CodeAnalysis.dll" goto :SkipAssembly
+    if "%%~nxF"=="Microsoft.CodeAnalysis.CSharp.dll" goto :SkipAssembly
+    if "%%~nxF"=="Microsoft.CodeAnalysis.VisualBasic.dll" goto :SkipAssembly
+    if "%%~nxF"=="Microsoft.Extensions.Options.ConfigurationExtensions.dll" goto :SkipAssembly
```

Comment:
Does this need links to issues that these are disabled against so that the exclusion is removed once the issue is fixed?

Reply:
Thanks for pointing that out. I have created and added mentions for the failures in <code>Microsoft.CodeAnalysis</code> and in <code>System.Runtime.WindowsRuntime</code>. The failures in <code>Microsoft.CodeAnalysis.CSharp</code> and <code>Microsoft.CodeAnalysis.VisualBasic</code> no longer repro so I removed them from the exclusion list. For the two <code>Microsoft.Extensions.Options...</code> assemblies, I marked them as "non-framework" as I believe that to match their status - e.g. I don't see them in my local Core_Root. Please let me know if you believe they merit further action.